src/compilers/template_engine.ts

```diff
@@ -34,17 +38,17 @@ export class TemplateEngine {
    * @param any data
    *     The data that should be rendered with the template.
    */
-  public render(template: string, data: any): string {
-    let code: any = "with(obj) { var r=[];\n";
-    let cursor: any = 0;
+  public render(template: string, data: unknown): string {
```

Comment:
Wonder if it's worth we give an example usage line as we can't properly type the `data` param?

Reply:
1a3e7f2